packages/react-components/react-nav-preview/src/components/NavItem/useNavItemStyles.styles.ts

```diff
@@ -1,19 +1,41 @@
-import { makeResetStyles, makeStyles, mergeClasses } from '@griffel/react';
+import { makeResetStyles, makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { NavItemSlots, NavItemState } from './NavItem.types';
-import { typographyStyles } from '@fluentui/react-theme';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
 
 export const navItemClassNames: SlotClassNames<NavItemSlots> = {
   root: 'fui-NavItem',
   content: 'fui-NavItem__content',
+  icon: 'fui-NavItem__icon',
 };
 
+// These should match the constants defined in @fluentui/react-icons
+// This package avoids taking a dependency on the icons package for only the constants.
+const iconClassNames = {
+  filled: 'fui-Icon-filled',
+  regular: 'fui-Icon-regular',
+};
```

Comment:
The icons will get tree-shaken, you could use this directly.

```ts
import { iconFilledClassName, iconRegularClassName } from '@fluentui/react-icons'; 
```

Reply:
updated and removed :-)